### DISCUSSION TITLE
In GUI have a separate view/totals for rejected deals

### DISCUSSION THREAD

**Discussion by stuberman** [filecoin-project/boost#556] [Ideas]
I am seeing many deal requests come in for zero price on unverified deals. These deals are rejected and counted as if they were actual deals. In reality these are rejected deal proposals. 

It would make sense to separate out the rejected proposals from actual deals that were either completed or errored for some reason. The deal counts should not include rejected deal proposals as they are different things.
<img width="222" alt="Boost deal counter" src="https://user-images.githubusercontent.com/49840642/171676685-99a34a6a-06f9-4c8a-a6ad-733ab96ff0a0.png">

Instead of **Storage Deals**
   Deals 123
   Legacy Deals 456789

It would be more useful to display something like:
**Storage Deals**
   Deals 123
   Legacy Deals 789
   Rejected Deals 456000

**Comment by LexLuthr** [comment]
@stuberman This is implemented in https://github.com/filecoin-project/boost/pull/551
